pkg/config/config_template.yaml

```diff
@@ -1452,6 +1452,14 @@ api_key:
   ##            collectors using the probabilistic sampler to ensure consistent sampling.
   #  hash_seed: 0
 
+  ## @param error_tracking_standalone - object - optional
+  ## Enables Error Tracking Standalone
+  ##
+  #error_tracking_standalone:
+  ## @env DD_APM_ERROR_TRACKING_STANDALONE_ENABLED - boolean - optional - default: false
+  ## Enables or disables Error Tracking Standalone
+  # enabled: false
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏You need to indent subkeys (see `internal_profiling.enabled` below for example)
```suggestion
  # error_tracking_standalone:
  #
    ## @param enabled - boolean - optional - default: false
    ## @env DD_APM_ERROR_TRACKING_STANDALONE_ENABLED - boolean - optional - default: false
    ## Enables or disables Error Tracking Standalone
    # enabled: false
```

Reply:
Addressed in https://github.com/DataDog/datadog-agent/pull/30065/commits/f62c51deb0fcc62ab5f6aded049aaaa24ca862ee

Based the work on what was done for the Probabilistic Sampler, but yes it does look clearer this way :)